build/package/Installer.MSI.targets

```diff
@@ -45,8 +45,7 @@
       <GenerateMsiVersion CommitCount="$(CommitCount)"
                           VersionMajor="$(VersionMajor)"
                           VersionMinor="$(VersionMinor)"
-                          VersionPatch="$(VersionPatch)"
-                          ReleaseSuffix="$(ReleaseSuffix)">
```

Comment:
It does use CommitCount, but that might not give a 100% unique version across branches with the same major.minor.path-commitcount